python/ray/tests/test_cli_patterns/test_ray_up_config.json

```diff
@@ -0,0 +1,51 @@
+{
+    "cluster_name": "test-cli",
+
+    "min_workers": 1,
+    "max_workers": 2,
+    "initial_workers": 1,
+
+    "target_utilization_fraction": 0.9,
+
```

Comment:
I've had cases where I made broad changes to cluster YAML files across the code-base. Can we change this to a yaml file just to avoid missing this?